creator-node/src/app.js

```diff
@@ -92,4 +98,44 @@ const initializeApp = (port, serviceRegistry) => {
   return { app: app, server: server }
 }
 
+const setupBullMonitoring = (serviceRegistry) => {
+  logger.info('Setting up Bull queue monitoring...')
+  const serverAdapter = new ExpressAdapter()
+  const {
+    snapbackSM,
+    syncQueue: syncProcessingQueueClass,
+    asyncProcessingQueue: asyncProcessingQueueClass,
+    monitoringQueue: monitoringQueueClass,
+    sessionExpirationQueue: sessionExpirationQueueClass,
+    skippedCIDsRetryQueue: skippedCidsRetryQueueClass
+  } = serviceRegistry
+  const { stateMachineQueue, manualSyncQueue, recurringSyncQueue } = snapbackSM
+  const { queue: syncProcessingQueue } = syncProcessingQueueClass
+  const { queue: asyncProcessingQueue } = asyncProcessingQueueClass
+  const { queue: imageProcessingQueue } = ImageProcessingQueue
+  const { queue: transcodingQueue } = TranscodingQueue
+  const { queue: monitoringQueue } = monitoringQueueClass
+  const { queue: sessionExpirationQueue } = sessionExpirationQueueClass
+  const { queue: skippedCidsRetryQueue } = skippedCidsRetryQueueClass
+
+  createBullBoard({
+    queues: [
+      new BullAdapter(stateMachineQueue, { readOnlyMode: true }),
+      new BullAdapter(manualSyncQueue, { readOnlyMode: true }),
+      new BullAdapter(recurringSyncQueue, { readOnlyMode: true }),
+      new BullAdapter(syncProcessingQueue, { readOnlyMode: true }),
+      new BullAdapter(asyncProcessingQueue, { readOnlyMode: true }),
+      new BullAdapter(imageProcessingQueue, { readOnlyMode: true }),
+      new BullAdapter(transcodingQueue, { readOnlyMode: true }),
+      new BullAdapter(monitoringQueue, { readOnlyMode: true }),
+      new BullAdapter(sessionExpirationQueue, { readOnlyMode: true }),
+      new BullAdapter(skippedCidsRetryQueue, { readOnlyMode: true })
+    ],
```

Comment:
i can't see queue state for all of these on the bull boards - can you?

i think that may have to do with the import order of the queues
rather than doing it here i think we should do it in the service registry, or someplace after all the queue objects have been created
i think if you require this before the queue class is initialized and instance is created then the variable here will be undefined? can you check that

Reply:
I've seen state for these before. I think you just don't see it because it removes the jobs after completion/failure. that said, I moved it into service registry anyway

Reply:
ahh ok, gotcha